pkg/controller/operators/olm/apiservices.go

```diff
@@ -277,6 +277,15 @@ func (a *Operator) areAPIServicesAvailable(csv *v1alpha1.ClusterServiceVersion)
 	return true, nil
 }
 
+func apiServiceDescriptionsForSSD(deploymentName string, descs []v1alpha1.APIServiceDescription) []v1alpha1.APIServiceDescription {
```

Comment:
What's SSD referring to? The function is straightforward but could benefit from some comments. Is it referring to strategyDetailsDeployment (SDD?)

Reply:
That is correct - I will make sure to improve function names and comments in the next update!